clients/drcachesim/tracer/output.cpp

```diff
@@ -82,9 +82,9 @@ get_local_window(per_thread_t *data)
 }
 
 static uint64
-local_instr_count_threshold()
+local_instr_count_threshold(uint64 trace_for_instrs)
 {
-    uint64 limit = op_trace_for_instrs.get_value();
+    uint64 limit = trace_for_instrs;
```

Comment:
We can do without this local copy `limit`.

Reply:
Done.